### ISSUE TITLE
Does finetune vit has a good performance？

### ISSUE THREAD

**Issue by henryccl** [notfolder/vit-pytorch#1]
Hello！I see this is code used to finetune vit in dogvscat dataset？And I’m trying to do the same thing, but my accuracy always get around 0.5（I just trained from the start,  non pre-trained model）， May I ask how about your result？
